### DISCUSSION TITLE
[VScode] Rome doesn't apply format on save if there's no rome.json in folder root

### DISCUSSION THREAD

**Discussion by JeViCo** [rome/tools#4703] [Q&A] [answered]
I was using prettier to format code on save. I switched to Rome VScode extension and noticed that format feature is disabled. It happened because of missing rome.json. Sadly enough, i switched back to prettier as it was essential for me when maintaining multiple projects.

Is this feature out yet? Am i doing something wrong?

**Comment by ematipico** [comment, marked as answer]
Untick this option and it will work 

https://docs.rome.tools/vscode/#romerequireconfiguration

**Reply by JeViCo** [reply]
Thank you, formatting works now! Am i able to define contents of rome.json globally without creating the file?

**Reply by ematipico** [reply]
Unfortunately, not yet. There's a proposal for that: https://github.com/rome/tools/discussions/4603
